packages/next/next-server/lib/router/utils/resolve-rewrites.ts

```diff
@@ -15,10 +17,17 @@ export default function resolveRewrites(
   resolveHref: (path: string) => string,
   locales?: string[]
 ) {
```

Comment:
As someone not-yet familiar with the Next.js codebase, I'd find it useful to have the return type specified here (now that it is not a primitive).

Reply:
Good point, added the return type here for better visibility 👍